lldb/packages/Python/lldbsuite/test/cpu_feature.py

```diff
@@ -0,0 +1,58 @@
+"""
+Platform-agnostic helper to query for CPU features.
+"""
+
+import re
+
+
+class CPUFeature:
+    def __init__(self, linux_cpu_info_flag: str, darwin_sysctl_key: str):
+        self.cpu_info_flag = linux_cpu_info_flag
+        self.sysctl_key = darwin_sysctl_key
```

Comment:
Judging from the instantiations at the bottom of the file, it would be better to set a default for the darwin key. Very few use it.